src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>1.4.0</VersionPrefix>
+    <VersionPrefix>1.5.0</VersionPrefix>
```

Comment:
Looking at the last few releases it seems more logical to match server version with the game version, but that's for future

Reply:
I'm not sure what you mean with this remark. While it might be useful to link release numbering to game versions, marketing versions and network broadcast versions rarely match each other. I don't think it is useful to change Impostor's versioning to match either:
-  If we match the marketing versions (what's on the main menu) that would mean that we must bump for every patch release including those that don't have or the latter and that an Impostor version would be compatible with versions even those that don't involve network changes
- If we match the network version (which is what the game identifies itself as), we have the issue that network versions don't clearly match marketing versions. For example, Marketing version 2021.6.15 uses Network version 2021.4.25, which is confusing as marketing version 2021.5.25.2 also exist, which uses a network version of 2021.4.2.

Current versioning scheme isn't perfect, but I haven't heard a good suggestion for a new one yet

Reply:
Fair enough, I see the issue.